org.eclipse.jdt.ls.core/src/org/eclipse/jdt/ls/core/internal/corext/template/java/CodeSnippetTemplate.java

```diff
@@ -69,13 +69,13 @@ public String getId() {
 	public static final String SYSERR_CONTENT = "System.err.println($${0});";
 	public static final String SYSTRACE_CONTENT = "System.out.println(\"${enclosing_type}.${enclosing_method}()\");";
 	public static final String FOREACH_CONTENT = "for ($${1:${iterable_type}} $${2:${iterable_element}} : $${3:${iterable}}) {\n" + "\t$${0}\n" + "}";
```

Comment:
I didn't add it because the template in `jdt.ui` do not have the selection: https://github.com/eclipse/eclipse.jdt.ui/blob/master/org.eclipse.jdt.ui/templates/default-templates.xml#L37-L39, though I don't know why either 🤔.

Do you need me to add `foreach` also?

Reply:
Yes please, so it's more consistent

Reply:
ok